apm-agent-plugins/apm-java-concurrent-plugin/src/main/java/co/elastic/apm/agent/concurrent/JavaConcurrent.java

```diff
@@ -58,6 +58,7 @@ public class JavaConcurrent {
         EXCLUDED_EXECUTABLE_TYPES.add("org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker");
         EXCLUDED_EXECUTABLE_TYPES.add("com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator");
         EXCLUDED_EXECUTABLE_TYPES.add("com.github.benmanes.caffeine.cache.BoundedLocalCache.PerformCleanupTask");
+        EXCLUDED_EXECUTABLE_TYPES.add("com.amazonaws.services.sqs.AmazonSQSAsyncClient$5");
```

Comment:
Good first check.
If this works, maybe we should change this set to contain wildcard matchers and in order to avoid the cost of matching every time an executable is submitted, maintain two list of already-encountered names - included and excluded. Because this nameless anonymous class name is probably not the most consistent thing to rely on.

Reply:
yes agreed, and it's more than the one nameless class in the outer class, several of which might apply